### PR TITLE
Responsive version

### PR DESCRIPTION
**Pull request by Somdotta07**
1.  Add a tags in HTML for header menu
2. Add div to Project and About me section
2. Add media queries to make the profile responsive
3. Adjust changes for desktop version
4. Add background and lines for desktop version
5. Change the layout of the projects 
6. Add styling to the whole page

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -483,3 +483,294 @@ hr {
...
+    border-bottom: none;
+  }
+
+  .contact-form .links {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    align-items: center;
+    margin-top: 100px;
+    border-top: 1px solid #dfe1e6;
```
Comment (dasileker) [inline style.css:766]: - [ ] I can't tell were the input is, please give them borders

File: style.css [hunk 2]
```diff
@@ -483,3 +483,294 @@ hr {
...
+  /* ---------------Programming Language------------ */
+
+  .prg {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    align-items: baseline;
+    margin-bottom: 120px;
+    width: 1024px;
+    margin-left: 200px;
+  }
```
Comment (dasileker) [inline style.css:731]: - [ ] The Icons beside the titles are far away, please make the margin of each be closer to the attributed title.

,,n,n

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @Somdotta07**, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Somdotta07) [reply style.css:766]: I followed the same template as figma. I'll give border.

File: style.css (hunk 2 above)
Comment (Somdotta07) [reply style.css:731]: Could you help me to understand how to fix that as  in live demo link the icons are close to the title.

Comment (dasileker) [reply style.css]: You can add margin.

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED ✔🎉**
Hello @Somdotta07 👋🏽
Your project is complete!  There is nothing else to say. It's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (Somdotta07) [reply style.css]: I didn't use icons separately, I used list-style-image for each icon

**Pull request merged**